chimedb/data_index/orm.py

```diff
@@ -658,6 +658,8 @@ class ArchiveFileCopyRequest(base_model):
     cancelled = pw.BooleanField(default=False)
     n_requests = pw.IntegerField()
     timestamp = pw.DateTimeField()
+    transfer_started = pw.DateTimeField(null=True)
+    transfer_completed = pw.DateTimeField(null=True)
 
-    class Meta(object):
-        primary_key = pw.CompositeKey("file", "group_to", "node_from")
+    class Meta:
+        indexes = ((("file", "group_to", "node_from"), False),)  # non-unique index
```

Comment:
It _really_ helps if there's a unique index on a table.  If this triad is no longer unique, can we at least add an auto_increment id?

Reply:
Specifically, what's preventing two rows from being identical?